### PR TITLE
adding QA tests for the new corgi UI

### PR DESCRIPTION
**Pull request by omehes**
These are the complete QA tests for the new corgi UI
Note, that the `@pytestrail.case("")` does not have a case number as the test cases are not yet updated in testrail. This will happen soon (I almost forgot about it)...

### CODE REVIEW

**Comment by omehes** [comment]
@openstax/ce-tech Integration tests failed due to independecy (playwright) issue... not sure...

**Review by therealmarv — APPROVED** [review]
looks very good for me :+1:

**Comment by omehes** [comment]
@therealmarv FYI: in case you come back to this PR, I added more commits...

**Review by TylerZeroMaster — CHANGES_REQUESTED** [review]
Other than these two small problems, this looks and works very well. If you want to resolve the merge conflict, you can just copy the circleci config from corgi-svelte onto this branch so that the contents match on both branches. I can do it if you would like.

File: backend/app/tests/ui/fixtures/ui.py [hunk 1]
```diff
@@ -19,4 +37,5 @@ def chrome_page():
 @pytest.fixture
 def corgi_base_url(base_url):
     """Return local corgi url"""
+    base_url = "https://corgi-514.ce.openstax.org/"
```
[the remark this answers is not in the document]
Comment (omehes) [reply backend/app/tests/ui/fixtures/ui.py:40]: Yes, you are right

File: backend/app/tests/ui/test_create_new_job_button.py [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+    # GIVEN: Playwright, chromium and the corgi_base_url
+
+    # WHEN: The Home page is fully loaded
+    chrome_page.goto(corgi_base_url)
+    home = HomeCorgi(chrome_page)
+
+    # WHEN: Input fields are filled and a job check box is selected
+    home.fill_repo_field(repo)
+    home.fill_book_field(book)
+    home.fill_version_field(book)
```
[the remark this answers is not in the document]
Comment (omehes) [reply backend/app/tests/ui/test_create_new_job_button.py:24]: oh, yes. thanks!!!

**Comment by omehes** [comment]
@TylerZeroMaster I made the changes (no conflict). I assume I can Squash and Merge my branch into corgi_svelte, right?

**Review by TylerZeroMaster — APPROVED** [review]

**Pull request merged**